test/fuzz/OracleTest.t.sol

```diff
@@ -530,6 +531,51 @@ contract OracleTest is Fixture, FunctionUtils {
         vm.stopPrank();
     }
 
+    /*//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+                                                        MORPHO                                                      
+    //////////////////////////////////////////////////////////////////////////////////////////////////////////////////*/
+
+    function testFuzz_ReadMorphoFeed(uint256[3] memory baseValues, uint256[3] memory normalizers) public {
```

Comment:
Let's do a test with a real Morpho feed as we never used them

Reply:
will be included when someone has deployed the steakUSDC/USD oracle in the script to check for USDA upgrade